src/Microsoft.AspNet.Mvc.ModelBinding/Binders/BindingSourceModelBinder.cs

```diff
@@ -59,6 +59,10 @@ protected BindingSourceModelBinder([NotNull] BindingSource bindingSource)
         /// <returns>
         /// A <see cref="Task"/> which will complete when model binding has completed.
         /// </returns>
+        /// <remarks>
+        /// Other model binders will never run if this method is called. Return <c>null</c> to skip other model binders
+        /// but allow higher-level handling e.g. falling back to empty prefix.
+        /// </remarks>
```

Comment:
This isn't true - when you return null other model binders will still run. 

What you're likely seeing is that our model binders are exclusive. If BodyModelBinder doesn't handle something nothing else will pick it up. However if you did this with something like ByteArrayModelBinder the collection model binder would at least _try_.